server.go

```diff
@@ -434,6 +434,16 @@ func (s *Server) Start() {
 		defer func() {
 			ConsumePanic(s.Sentry, s.Statsd, s.Hostname, recover())
 		}()
+
+		// We want to align our ticker to a multiple of it's duration for
+		// convenience of bucketing.
+		time.Sleep(s.CalculateTickDelay(time.Now()))
```

Comment:
Per slack, `<-time.After` is marginally better, although not terribly consequential either way